doc/source/cd/background.ipynb

```diff
@@ -0,0 +1,608 @@
+      "text/plain": [
+       "<Figure size 288x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Mean test accuracy 66.67%\n",
+      "Is drift? Yes!\n"
+     ]
+    }
+   ],
+   "source": [
+    "slope = -1.0\n",
+    "\n",
+    "shift_norm_0 = sp.stats.multivariate_normal([2, -4], np.eye(2)*sigma**2)\n",
+    "X_0 = shift_norm_0.rvs(size=60,random_state=2)\n",
+    "X_1 = train_norm_1.rvs(size=60,random_state=2)\n",
+    "X_test = np.vstack([X_0, X_1])\n",
+    "\n",
+    "# Plot\n",
+    "y_test = true_model(X_test,slope)\n",
+    "plot(X_test,y_test,slope,clf=clf)\n",
+    "\n",
+    "# Classifier accuracy\n",
+    "print('Mean test accuracy %.2f%%' %(100*clf.score(X_test,y_test)))\n",
+    "\n",
+    "# Check for drift in covariates\n",
+    "pred = detector.predict(X_test)\n",
+    "print('Is drift? %s!' %labels[pred['data']['is_drift']])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The test data has drifted into a previously unseen region of feature space, and the model is now misclassifying a number of test observations. If true test labels are avaiable, this is easily detectable by monitoring the test accuracy. However, labels aren't always avaiable during deployment, in which case a drift detector monitoring the covariates comes in handy. In this case, the trained MMD detector successfully detects the covariate drift.\n",
+    "\n",
+    "In a similar manner, prior drift can be monitored by training a detector on training labels, and then feeding it a model's predicted labels:\n",
```

Comment:
Being a bit cautious here -> a proxy for prior drift + the usual `y_train` -> `y_ref`.